Shield/App.xaml.cs

```diff
@@ -55,6 +56,13 @@ sealed partial class App : Application
         /// </summary>
         public App()
         {
+            WindowsAppInitializer.InitializeAsync()
```

Comment:
probably should be try...catched so that a bad telemetry component/connection won't take down the app